wavesexchange_topic/src/lib.rs

```diff
@@ -116,12 +118,20 @@ pub struct LeasingBalance {
     pub address: String,
 }
 
+#[derive(Clone, PartialEq, Eq, Hash, Debug)]
+pub struct ExchangePairs {
```

Comment:
Bad name for the type. It represents a single pair, so it should be named `ExchangePair` instead.